src/recommendations/RecommendationsList.jsx

```diff
@@ -5,19 +5,19 @@ import PropTypes from 'prop-types';
 import ProductCard from './ProductCard';
 
 const RecommendationsList = (props) => {
-  const { recommendations, userId } = props;
+  const { recommendations, userId, isLoading } = props;
 
   return (
-    <div className="d-flex recommendations-container__card-list">
+    <div className="d-flex flex-wrap mb-3 recommendations-container__card-list">
       {
         recommendations.map((recommendation, idx) => (
-          <span key={recommendation.uuid}>
-            <ProductCard
-              product={recommendation}
-              position={idx}
-              userId={userId}
-            />
-          </span>
+          <ProductCard
+            key={recommendation.uuid}
```

Comment:
Are we getting `uuid` from Algolia index?

Reply:
yes we are